joins.sql

```diff
@@ -0,0 +1,85 @@
+-- STEP 1
+-- SELECT *
+-- FROM users;
+
+-- STEP 2
+-- SELECT *
+-- FROM posts
+-- WHERE users_id = 100;
+
+-- STEP 3
+-- SELECT p.id, p.title, p.url, p.content, p.created_at, p.updated_at, u.first_name, u.last_name
+-- FROM posts p, users u
+-- WHERE u.id = p.users_id AND u.id = 200;
+
+-- STEP 4
+-- SELECT p.id, p.title, p.url, p.content, p.created_at, p.updated_at, u.username
+-- FROM posts p, users u
+-- WHERE u.first_name = 'Norene' AND u.last_name = 'Schmitt';
+
+-- STEP 5
+-- SELECT u.username, p.created_at
+-- FROM users u LEFT JOIN posts p ON u.id = p.id
+-- WHERE p.created_at >=  '2015-01-01'
+-- ORDER BY p.created_at;
+
+-- STEP 6
+-- SELECT p.title, p.content, u.username
+-- FROM users u RIGHT JOIN posts p ON u.id = p.users_id
+-- WHERE u.created_at < '2015-01-01'
+-- ORDER BY u.created_at DESC;
+
+-- STEP 7
+-- SELECT p.title AS "Post Title", c.id, c.body, c.created_at, c.updated_at, c.users_id, c.posts_id
+-- FROM posts p RIGHT JOIN comments c ON p.id = c.posts_id;
```

Comment:
`JOIN`, `INNER JOIN`, and `LEFT JOIN` are more common and should be used when possible over `RIGHT JOIN`. Your code will be read/used by others and should be as easy to follow as possible.